.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:react/recommended",
+    "plugin:jsx-a11y/recommended",
+    "prettier",
+    "prettier/react"
+  ],
+  "rules": {
+    "react/prop-types": 0,
+    "no-console": 1
```

Comment:
I would recommend adding these extra rules:
` "no-unused-vars": [
            "error",
            {
                "argsIgnorePattern": "^_",
                "varsIgnorePattern": "^_",
                "ignoreRestSiblings": true
            }
        ],
        "no-unexpected-multiline": "off",
        "no-console": ["error", { "allow": ["info", "warn", "error"] }],
        "react/prop-types": [
            "error",
            {
                "skipUndeclared": true
            }
        ],
        "prettier/prettier": [
            "error",
            {
                "singleQuote": true,
                "trailingComma": "es5",
                "tabWidth": 4,
                "useTabs": false
            }
        ]`

Reply:
I have added these rules, looks like they're not compatible with my prettierrc
![image](https://user-images.githubusercontent.com/36889080/69453063-6a072d00-0d63-11ea-997c-87a9ec6db7b2.png)
please provide this file aswell.

Reply:
Changes made - with exceptions:
no-console stays at 1 - no need to display as error
singleQuotes stays at false - I prefer double quotes